tests/cc13b/input.dat

```diff
@@ -0,0 +1,50 @@
+#! Tests RHF CCSD(T)gradients
+memory 500 mb
+
+molecule h2o {
+0 1
+O
+H 1 R
+H 1 R 2 A
+
+R = 1.0
+A = 104.5
+}
+
+set {
+   reference rhf
+   basis cc-pvdz
+   r_convergence 13
+   e_convergence 13
+   d_convergence 13
+}
+
+analytic, wfn = gradient('ccsd(t)', return_wfn=True)
+
+set findif points 5
+fin_diff, wfn = gradient('ccsd(t)', dertype=0, return_wfn=True)
```

Comment:
This test doesn't take very long to run (1-2 min on 1 thread). I wonder if we could hard-code in the results from finite differences in for the comparison, and possibly move this to quick-tests?

I could see RHF-CCSD(T) gradients being a popular feature and it would be a good idea to make sure it hasn't broken.